packages/hoprd/crates/Makefile

```diff
@@ -17,12 +17,13 @@ $(JS_FILES):
 	@echo "--- Building WASM module to '$(@D)' ... ---"
 	mkdir -p $(@D)
 	cd $(@D)/.. && wasm-pack build --target=nodejs
+	cp $@ $(subst .js,.cjs,$@)
 	@echo "--- Built WASM module in '$(@D)' ---"
 
 install:
 ifneq ($(PACKAGES),)
 	@mkdir -p ../lib
-	install $(foreach pkg, $(PKG_DIRS), $(pkg)/*.js) ../lib/
+	install $(foreach pkg, $(PKG_DIRS), $(pkg)/*.cjs) ../lib/
 	install $(foreach pkg, $(PKG_DIRS), $(pkg)/*.ts) ../lib/
```

Comment:
Shouldn't it say `*.d.ts` - Guess that we're only copying *types*

Reply:
Yes, the `wasm-pack` generates `.d.ts` files only, so this in fact copies all the `.d.ts` files.